connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/DB2IntegrationSuite.scala

```diff
@@ -61,9 +61,6 @@ class DB2IntegrationSuite extends DockerJDBCIntegrationV2Suite with V2JDBCTest {
   override def sparkConf: SparkConf = super.sparkConf
     .set("spark.sql.catalog.db2", classOf[JDBCTableCatalog].getName)
     .set("spark.sql.catalog.db2.url", db.getJdbcUrl(dockerIp, externalPort))
-    .set("spark.sql.catalog.db2.pushDownAggregate", "true")
-    .set("spark.sql.catalog.db2.pushDownLimit", "true")
-    .set("spark.sql.catalog.db2.pushDownOffset", "true")
```

Comment:
Could you revert this? Usually, we keep these explicitly because this is a kind of prerequisite of the test case. It's also good for code readability.